README.md

```diff
@@ -1,6 +1,12 @@
-# Node.js
-
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/nodejs/node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/29/badge)](https://bestpractices.coreinfrastructure.org/projects/29)
+<p align="center">
+  <a href="https://nodejs.org/">
+    <img src="https://nodejs.org/static/images/logo-light.svg" width="400" />
```

Comment:
I think I'll just go with `alt="Node.js"` and remove the link on the logo as there are already multiple links in the document that link to the homepage.